hazelcast/src/main/java/com/hazelcast/core/IMap.java

```diff
@@ -149,6 +156,24 @@
      * <p>
      * If you have previously set a TTL for the key, the TTL remains unchanged and the entry will
      * expire when the initial TTL has elapsed.
+     *
+     * <p><b>Interactions with the data store</b>
```

Comment:
can we call it as `map store` instead of `data store`?

Reply:
Renamed to `map store`.